src/style.css

```diff
@@ -1,3 +1,73 @@
+:root {
+  --logo-width: 40px;
+  --logo-height: 40px;
+  --nav-height: 60px;
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 0 50px;
+}
+
+.header a.logo {
+  width: var(--logo-width);
+  height: var(--logo-height);
+}
+
+.nav ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  align-items: center;
+}
+
+.nav ul li {
+  margin: 0;
+  padding: 0;
+}
+
+.nav ul li:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav ul li a {
```

Comment:
Cassia, it is much better to assign classes and style those classes instead of doing like this.
Because this is used to override styling when needed through specificity.